### ISSUE TITLE
Time manager don't switch layer for multiple layers

### ISSUE THREAD

**Issue by petunjiaj** [3liz/lizmap-web-client#1239]
I to all, I'm try to modify timemanager for little customization.
I found that, when there are more than one layer in tmLayers config file (of project), It load them one-by-one, then it sets slider and attributes only for last loaded layer.
Moreover, setting only last loaded layer, it change order everytime from loading results: sometime it sets one layer, sometimes another.

I'm not a js expert, so I tried to swith them following a selector-box and reload layer by calling setAnimationBoundariesFromLayer(aName) function..
clearly I missed something so the layer doesn't match and has no features
(error for undefined features => Uncaught TypeError: minTime.getTime is not a function)

A part of this, I'm asking if there is a way to switch (in timelayer) for playing every layer setted in time-manager config file of a project.
If I could understand how to do it, I could set layer by my custom select-box (choosing witch layer I want to play)
tank you.

**Comment by saeri-ims** [comment]
Hello similarly to @petunjiaj i am not a js expert but the point made in the question is interesting. It would be great to have the opportunity to add more than one layer to the time manager. My current workaround has been to merge all the layers (luckily representing the same "object") and add the merged layer to the time manager tool. 
Knowing a bit more how to customise the time manager tool perhaps can help, also to see if it is possible to define a start and end time for each layer (see this request too: https://github.com/3liz/lizmap-web-client/issues/353). Anyway, thanks a lot to what Lizmap dev are doing as the service provided is altogether very very goodl!!

**Comment by Gustry** [comment]
Did you get some news about your experiments? It looks like a nice feature request if I understand correctly.

**Comment by petunjiaj** [comment]
I recently resumed my job. 
I found that my script (some upgrades on time-manager.js) is not working anymore with new versions of lizmap.
I will try to resolve this problem in next months. I'll keep you up to date.

**Comment by petunjiaj** [comment]
I made some little upgrades, but I can't understand some code parts..
When I switch from one layer to another I can set layer visibility (setvisibility()) and get right layer highlighted:
(the first point of layer is highlighted).
To setup layer with slider and points I should call two functions

- setAnimationBoundariesFromLayer(layer.name)
- setLayerStyleMap(layer.name)

But, when I try to call them I have no effects.
I found that the problem should be on layer.features.

If I try to get layer.features from loaded layer (tmLayerConfig.layer.features) they are empty.
If I load again the layer (layer.events.on({ loadend: function(evt) {...}}); ) I have features, with many initialize, but in console they don't show layer info (length is 0) instead of what happen at minidock opening.

I don't understand very much in scripting, I need some help..
Tank you to everyone should help me.
